.helm/ecamp3/values.yaml

```
@@ -15,6 +15,7 @@ helpLink: # 'https://ecamp3.ch/faq'
 # enable/disable feature across the complete deployment
 featureToggle:
   developer: false # enables various tools/features foreseen for development deployments (language switcher, form controls view, performance measurement view, etc.)
+  checklist: false # enables various tools/features foreseen for development deployments (language switcher, form controls view, performance measurement view, etc.)
```

```suggestion
  checklist: false
```
the comment is wrong.
thankfully, you did not attach the feature to the developer flag